colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,984 @@
+{
```

Comment:
<ul><li>Perhaps change the ORG_NAME to &lt;ENTER YOUR ORG_NAME HERE&gt; (or something along those lines)</li><li>VERESION = 0 -&gt; "VERSION"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/560/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>